ceph/rados/core_workflows.py

```diff
@@ -1202,6 +1201,147 @@ def delete_pool(self, pool: str) -> bool:
         log.error(f"Pool:{pool} could not be deleted on cluster")
         return False
 
+    def create_fragmented_osds_on_pool(
+        self,
+        pool: str = None,
+        required_fragmentation: float = 0.03,
+        timeout: int = 12000,
+        num_threads: int = 25,
+    ) -> bool:
+        """
+        Method to add fragmentation on OSDs of the cluster by writing objects at various offsets. This method writes
+        500 objects with offsets in each iteration and checks if desired level of fragmentation is achieved.
+
+        Args:
+            pool: name of the pool on which the Objects with offset need to be written
+            required_fragmentation: desired level of fragmentation on the cluster to be reached
+            timeout: timeout for the method
+            num_threads: number of threads to use. Note: Too many can cause host OOM kill
+
+        Return:
+            True -> Desired level of fragmentation achieved on the cluster
+            False -> Desired level of fragmentation not achieved on the cluster
+        """
+
+        def create_objects(pool_name, obj_filename, start, end) -> bool:
+            """
+            Writes objects at different offsets to introduce fragmentation.
+            """
+            for obj in range(start, end):
+                for i in range(0, 15 * 60 * 1024, 60 * 1024):
```

Comment:
I think this should be changed to `for i in range(0, 60 * 1024, 15 * 60 * 1024)`

Reply:
it's correct.